drivers/espi/espi_emul.c

```diff
@@ -113,7 +113,7 @@ static int espi_emul_read_lpc_request(const struct device *dev,
 #ifdef CONFIG_ESPI_PERIPHERAL_ACPI_SHM_REGION
 	case EACPI_GET_SHARED_MEMORY:
 		__ASSERT_NO_MSG(api->get_acpi_shm);
-		*data = (uint32_t)api->get_acpi_shm(emul);
+		*data = (uint32_t)api->get_acpi_shm(emul->parent, emul);
```

Comment:
I find the use of `parent` here confusing, when the function prototype uses `target` for this parameter.